rosbag2_storage_default_plugins/include/rosbag2_storage_default_plugins/sqlite/sqlite_storage.hpp

```diff
@@ -79,12 +82,17 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteStorage
 
   std::shared_ptr<SqliteWrapper> database_;
   std::string database_name_;
+  std::vector<std::string> database_files_;
```

Comment:
`database_files_` and `database_name_` are containing the same information. Having denormalized information there is a risk for this class. Instead we should replace each use of `database_name_` by a call to `get_current_database_file_name()` implemented as follow:

```c++
std::string get_current_database_file_name() const {
  return database_files_.back();
}
```

This will naturally remove potential bugs where `database_name_ != database_files_back()`.